### ISSUE TITLE
[BUG] browserslist: Unknown arguments --hot.

### ISSUE THREAD

**Issue by Satawe-no** [Dabolus/pwa-webpack-starter-kit#2]
Hi,

The project looks amazing, I'de like to play with it, but I get some errors with scripts in package.json.

**Describe the bug**
I get this error `browserslist: Unknown arguments --hot.` when I run `yarn run start`. Maybe I have to do something with my windows 10 os ? If I remove this `BROWSERSLIST=\"edge > 12\"` it works but I get another error in the browser console `__webpack_require__(...) is not a function`. Should I do something to make it works?

**To Reproduce**
1. run script: `yarn run start`

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Chrome 
 - Version 70.0.3538.110

**Comment by Dabolus** [comment]
Hi! Thanks for you interest in this project. Assigning environment variables in that way only works on bash-like shells, and that's the reason why you are experimenting this issue on Windows. It is quite easy to make it cross-platform, though. Give me a couple of minutes.

**Comment by Dabolus** [comment]
@Satawe-no the issue should be fixed now, can you please pull the master branch and check if it works?

**Comment by Satawe-no** [comment]
Thank you 👍 

But i'm still with the other problem `__webpack_require__(...) is not a function`. You don't have this problem ?

**Comment by Dabolus** [comment]
After finally putting my hands on a Windows PC I was able to reproduce the issue. Seems like some Webpack plugin is causing this, as I'm able to get the error even with an ultra simple app with the same Webpack configuration. I will investigate more deeply in the next few days and check back when I manage to find a solution

**Comment by Satawe-no** [comment]
Great, I hope you will find the fix fastly, I'm looking forward to trying ! I try on my side too to find a solution.

**Comment by benbenbenbenbenben** [comment]
I don't know if this is a complete fix, but I changed `base.config.ts` line 41 so that it matches both back slash and forward slash and now `yarn run start` works for me.

```javascript
exclude: /node_modules[\/\\](?!(pwa-helpers|@polymer)[\/\\]).*/,
```

**Comment by Satawe-no** [comment]
@benbenbenbenbenben Works for me too with this code. Thanks

**Comment by Dabolus** [comment]
@benbenbenbenbenben great catch, thanks!

**Comment by Dabolus** [comment]
Fixed by merging #3
